CONTRIBUTING.md

```diff
@@ -0,0 +1,25 @@
+# Contributing to Fetch.ai ledger
+
+:+1::tada: First off, thanks for taking the time to contribute! :tada::+1:
+
+The following is a set of guidelines for contributing to the [Fetch.ai ledger repository](https://github.com/fetchai/ledger) on GitHub. We strongly encourage you to follow them as closely as possible; if you are dealing with a case not specified here, use your best judgment and feel free to propose changes to the document in a pull request.
+
+
+## Bug reports
+
+Before creating a bug report, please check [this list](https://github.com/fetchai/ledger/issues) as you might find out that you don't need to create one. When you are creating a bug report, please include as many details as possible.
+
+**Note:** If you find a **Closed** issue that seems like what you're experiencing, open a new issue and include a link to the original issue in the body of your new one.
+
+
+## Code contributions
```

Comment:
It is definetly worth mentioning that we have strict style checkers, you need to run apply_style.py at the very least to get it to pass CI

Reply:
It also adds the copyright header that is required for all files